Ginger/Ginger/RunSetLib/CreateAutoRunWizardLib/AutoRunWizardCLITypePage.xaml.cs

```diff
@@ -72,15 +115,19 @@ private void ShowHelp()
             }*/
             if (xDynamicRadioButton.IsChecked == true)
             {
-                helpContent = string.Format("File (XML/JSON) which describes the {0} to be executed." + GetRowDown() + "To be used in case {0} not exist in the Solution and should be created dynamically for execution purposes only, or in case {0} is exist but need to dynamically customized it input values for specific execution." + GetRowDown() + "Executed by triggering Ginger executer with the argument 'Dynamic -f %FilePath%', Example: Ginger.exe Dynamic -f \"C:\\Ginger\\FeatureATesting.Ginger.AutoRunConfigs.xml\"", GingerDicser.GetTermResValue(eTermResKey.RunSet));
+                helpContent = "Execute customized or virtual Run set via configuration file.";
             }
             else if (xScriptRadioButton.IsChecked == true)
```

Comment:
code of all not supported options like xScriptRadioButton should be removed from xaml and cs, need to keep only code related to 3 options we show on UI args/dynamic/request